test/e2e/examples_test.go

```diff
@@ -112,12 +113,56 @@ func TestCppSimpleGameServerReady(t *testing.T) {
 				ContainerPort: 7654,
 				Protocol:      corev1.ProtocolUDP,
 			}},
+			Health: agonesv1.Health{
+				InitialDelaySeconds: 300,
+				PeriodSeconds:       25,
+			},
```

Comment:
Can you do me a favour - can we remove the Health settings, and we can test without it?

```suggestion
```

We shouldn't have to make them so long now, since it looks like it was a resource setting issue that we fixed below.

Reply:
Removing this health settings indicated that the GameServer reached a terminal 'Unhealthy' state upon testing- https://gist.github.com/Kalaiselvi84/8d7a212a93c04291d42f90476c0fcb7f

Reply:
Even with the resource settings?

Reply:
Yep! it fails with resource and without health.

Reply:
Oh yeah it would! Because it does't make health calls very often!

Okay, let's try a small number - probably:

```go
			Health: agonesv1.Health{
				InitialDelaySeconds: 30,
				PeriodSeconds:       30,
			},
```

Should be a good fit.